### PR TITLE
Fixes for the official build.

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
Update build tools to pick up a fix to use the production Helix API.
The Compression and SqlClient packages will pick up a few runtime packages
that are built in TFS, allowing us to build and ship the majority of their
content from the open.

### CODE REVIEW

File: src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj [hunk 1]
```diff
@@ -10,6 +10,14 @@
...
   <ItemGroup>
+    <RuntimeDependency Include="runtime.win7-amd64.runtime.native.System.Data.SqlClient.sni">
+      <TargetRuntime>win7-amd64</TargetRuntime>
+      <Version>4.0.1-$(ExternalExpectedPrerelease)</Version>
+    </RuntimeDependency>
+    <RuntimeDependency Include="runtime.win7-x86.runtime.native.System.Data.SqlClient.sni">
+      <TargetRuntime>win7-x86</TargetRuntime>
+      <Version>4.0.1-$(ExternalExpectedPrerelease)</Version>
+    </RuntimeDependency>
     <ProjectReference Include="win\runtime.native.System.Data.SqlClient.sni.pkgproj">
```
Comment (ericstj) [inline src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj]: Should you remove these project references?

Comment (weshaggard) [inline src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj]: Yes at least comment them out until we can get them building in corefx.

Comment (jhendrixMSFT) [inline src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj]: Good catch, I have removed them.

**Comment by ericstj** [comment]
LGTM

File: src/Native/pkg/runtime.native.System.IO.Compression/runtime.native.System.IO.Compression.pkgproj [hunk 2]
```diff
@@ -43,15 +55,6 @@
     <ProjectReference Include="win8\runtime.native.System.IO.Compression.pkgproj">
       <Platform>arm</Platform>
     </ProjectReference>
-    <ProjectReference Include="win10\runtime.native.System.IO.Compression.pkgproj">
```
Comment (ianhays) [inline src/Native/pkg/runtime.native.System.IO.Compression/runtime.native.System.IO.Compression.pkgproj]: or does it only reference the endpoints i.e. the pkgproj files that specify the NativePath?

Comment (ianhays) [inline src/Native/pkg/runtime.native.System.IO.Compression/runtime.native.System.IO.Compression.pkgproj]: nevermind. I forgot about your accompanying TFS checkin that fixes that.

Comment (jhendrixMSFT) [inline src/Native/pkg/runtime.native.System.IO.Compression/runtime.native.System.IO.Compression.pkgproj]: It references the leaf pkgproj files.

**Comment by ianhays** [comment]
LGTM

**Pull request merged**